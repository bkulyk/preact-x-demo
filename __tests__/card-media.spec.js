import React from 'react';
import CardMedia from '@bitchin/card-media';
import { shallow } from './enzyme';
import { it, expect } from './jest';

it('renders correctly, with properties', () => {
  expect(shallow((
    <CardMedia
      title="some-title"
      url="http://example.com/awesome-pic.png"
    />
  ))).toMatchSnapshot();
});

it('renders correctly, with different props', () => {
  expect(shallow((
    <CardMedia
      url="http://example.com/awesome-pic.png"
      ratio="square"
    />
  ))).toMatchSnapshot();
});
