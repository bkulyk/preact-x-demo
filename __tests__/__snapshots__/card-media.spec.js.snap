// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly, with different props 1`] = `
<div
  className="mdc-card__primary-action"
>
  <div
    className="mdc-card__media mdc-card__media--square"
    style={
      Object {
        "backgroundImage": "url(http://example.com/awesome-pic.png)",
      }
    }
  >
    <div
      className="mdc-card__media-content"
    />
  </div>
</div>
`;

exports[`renders correctly, with properties 1`] = `
<div
  className="mdc-card__primary-action"
>
  <div
    className="mdc-card__media mdc-card__media--16-9"
    style={
      Object {
        "backgroundImage": "url(http://example.com/awesome-pic.png)",
      }
    }
  >
    <div
      className="mdc-card__media-content"
    >
      some-title
    </div>
  </div>
</div>
`;
